src/schema/v2/partner/partner.ts

```diff
@@ -367,6 +386,58 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           })
         },
       },
+      partnerAlertsHitsConnection: {
```

Comment:
nit: I would put the plural only on Hit so what about `partnerAlertHitsConnection`?